src/main/scala/org/antipathy/scoozie/action/HasPrepare.scala

```diff
@@ -0,0 +1,28 @@
+// $COVERAGE-OFF$
+package org.antipathy.scoozie.action
+import scala.collection.immutable.Map
+import org.antipathy.scoozie.action.prepare.Prepare
+import org.antipathy.scoozie.configuration.ActionProperties
+
+/**
+  * Trait for building prepare properties
+  */
+private[scoozie] trait HasPrepare {
+  this: Nameable =>
+  import scala.xml.Elem
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Put imports in the beginning of the package](https://app.codacy.com/app/SimonJPegg/scoozie/pullRequest?prid=3078561)